system-tests/services/test-apps.js

```diff
@@ -938,9 +1269,79 @@ describe("Services", function() {
         .getTableRowThatContains(serviceName)
         .get(".bar.healthy", { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
         .should("exist");
+
+      // Now click on the name
+      cy
+        .get(".page-body-content table")
+        .getTableRowThatContains(serviceName)
+        .get("a.table-cell-link-primary")
+        .contains(`${serviceName}`)
```

Comment:
you can always use the reference `serviceName ` instead template strings. Not a big deal though.